code/modules/mob/living/living.dm

```diff
@@ -474,31 +474,39 @@
 			if(MOVE_INTENT_WALK)
 				. += config.walk_speed
 
-/mob/living/proc/makeTrail(turf/target_turf)
+/mob/living/proc/makeTrail(turf/target_turf, soft_crit_override_dir = null)
```

Comment:
this shouldn't mention soft crit.
what's the point in a generic argument if it still contains its original snowflake usage in its name.